Tasks/XamariniOS/xamarinios.ts

```diff
@@ -14,30 +14,31 @@ var args = tl.getInput('args');
 var packageApp = tl.getBoolInput('packageApp');
 var buildForSimulator = tl.getBoolInput('forSimulator');
 var device = (buildForSimulator) ? 'iPhoneSimulator' : 'iPhone';
-tl.debug('Device: ' + device);
+var xbuildLocation = tl.getInput('mdtoolLocation', false);
+var cwd = tl.getInput('cwd');
+tl.debug('solution: ' + solutionPath);
+tl.debug('configuration: ' + configuration);
```

Comment:
I specifically removed these when I last touched this task. There was feedback on another thread that we should remove these since task-lib handles them